sway/commands/output.c

```diff
@@ -20,6 +20,22 @@ static char *bg_options[] = {
 	"tile",
 };
 
+static struct cmd_results *cmd_output_dpms(struct output_config *output,
+		int *i, int argc, char **argv) {
+
+	if (++*i >= argc) {
+		return cmd_results_new(CMD_INVALID, "output", "Missing dpms argument.");
+	}
+
+	char *value = argv[*i];
+	if (strcmp(value, "on") == 0) {
+		output->dpms_state = DPMS_ON;
+	} else if (strcmp(value, "off") == 0) {
+		output->dpms_state = DPMS_OFF;
+	}
```

Comment:
Can you return an error if it's neither "on" nor "off"?

Reply:
Done